attending/doc.py

```diff
@@ -4,6 +4,12 @@
 import os
 
 
+def get_module_version(module):
+    if hasattr(module, "__version__"):
+        return module.__version__
+    return "latest"
```

Comment:
can you add the `version = pkg_resources.working_set.by_key[mod_name].version` logic to this, so these installed packages report the version installed.